src/SignalR/clients/ts/FunctionalTests/package.json

```diff
@@ -30,9 +30,11 @@
     "karma-sauce-launcher": "^1.2.0",
     "karma-sourcemap-loader": "^0.3.7",
     "karma-summary-reporter": "^1.6.0",
+    "process": "^0.11.10",
     "rxjs": "^6.3.3",
     "ts-node": "^8.6.2",
     "typescript": "^3.7.5",
+    "webpack": "^5.23.0",
```

Comment:
This is nice, didn't you get into issues with common pollyfills not being included anymore?

I did started down this path but somehow I produced a bigger bundle for Blazor than the one we currently produce.

Reply:
Yes; it was just `process` in the tests, though is more problematic with the blazor js (as that's missing more pollyfills)